js/models/fixed_page_spread.js

```diff
@@ -133,6 +193,12 @@ var Spread = function(spine, isSyntheticSpread) {
         return Spread.POSITION_CENTER;
     }
 
+    /**
+     * Sets an item to be opened next
```

Comment:
for consistency with openPrev(): "Opens the next item" (or vice-versa, but needs to be consistent)